spec/integration/active_support/Gemfile

```diff
@@ -0,0 +1,15 @@
+source 'http://rubygems.org'
+
+gem 'hashie', path: '../../..'
+require File.expand_path('../../../../lib/hashie/extensions/ruby_version', __FILE__)
```

Comment:
For next time, if you feel super motivated I made https://github.com/dblock/with-version